.github/workflows/golangci-test.yml

```diff
@@ -0,0 +1,24 @@
+name: golangci-test
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - develop
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.14'
+      - name: Run coverage
+        run: go test -coverprofile=coverage.out -covermode=atomic
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v2
+        files: ./coverage.out
```

Comment:
There are some errors at [this line](https://github.com/openebs/openebsctl/actions/runs/1181384766/workflow) PTAL, I don't think a `CODECOV_TOKEN` is required to be set as this is a public repository.

I think, it should be `file` inside the `with` object/key, like [this](https://github.com/codecov/codecov-action/blob/master/.github/workflows/main.yml#L22)

Reply:
How is the codecov token configured?

Reply:
Per the docs, CODECOV_TOKEN [isn't required](https://github.com/codecov/codecov-action#usage) for public repositories if the CI is GitHub actions, CircleCI, etc.